src/App.tsx

```diff
@@ -21,75 +21,110 @@ enum SortType {
   LENGTH,
 }
 
-type ReorderOptions = {
+type State = {
   sortType: SortType,
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+export class App extends React.Component<{}, State> {
+  state = {
+    sortType: 0,
```

Comment:
You have an Enum with sort types. Use it, why do you need to hardcore it with numbers?